### PR TITLE
specify react-native-web version

### PR DESCRIPTION
**Pull request by chrisdrackett**
my understanding is that >-0.13 is not yet supported

# Why

prevent issues around using incorrect version of RNW

### CODE REVIEW

File: docs/pages/guides/running-in-the-browser.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ As of SDK 33 you can use Expo to create web apps that run in the browser using t
 Starting in _SDK 33_ projects bootstrapped with the Expo CLI will have web support from the start. To add web support to an existing Expo app you can do the following:
 
 - Install the latest version of the Expo CLI: `npm i -g expo-cli`
-- Add web dependencies: `yarn add react-native-web react-dom`
+- Add web dependencies: `yarn add react-native-web@~0.12 react-dom`
```
Comment (EvanBacon) [inline docs/pages/guides/running-in-the-browser.md:14]:
```suggestion
- Add web dependencies: `yarn add react-native-web@~0.11 react-dom`
```

File: docs/pages/guides/running-in-the-browser.md [hunk 2]
```diff
@@ -37,7 +37,7 @@ The Expo Unimodules and dev-tools are highly composable and can be used in _any_
  Here is how to use the web portion of the modules in a project bootstrapped with either the React Native CLI (`react-native init`) or Create React App ([`create-react-app`](https://github.com/facebook/create-react-app)).
 
 - Install the latest version of the Expo CLI: `npm i -g expo-cli`
-- Add web dependencies: `yarn add react-native-web react-dom expo`
+- Add web dependencies: `yarn add react-native-web@~0.12 react-dom expo`
```
Comment (EvanBacon) [inline docs/pages/guides/running-in-the-browser.md:40]:
```suggestion
- Add web dependencies: `yarn add react-native-web@~0.11 react-dom expo`
```

**Review by EvanBacon — APPROVED** [review]
ok

**Comment by zhigang1992** [comment]
Should be perhaps be `expo install react-native-web` instead?

Do we have any more context why 13 isn't compatible?

**Pull request merged**